awx/main/tasks.py

```diff
@@ -1146,6 +1149,36 @@ def run(self, pk, **kwargs):
         '''
         Run the job/task and capture its output.
         '''
+        self.instance = self.model.objects.get(pk=pk)
+        containerized = self.instance.is_containerized
+        pod_manager = None
+        if containerized:
+            # TODO: Document what's going on here
+            from awx.main.scheduler.kubernetes import PodManager # Avoid circular import
+            pod_manager = PodManager(self.instance)
+            self.cleanup_paths.append(pod_manager.kube_config)
+            try:
+                pod_manager.deploy()
+            except (ApiException, Exception) as exc:
+                if isinstance(exc, ApiException) and exc.status == 403:
+                    try:
+                        if 'exceeded quota' in json.loads(exc.body)['message']:
+                            logger.warn(exc.body)
+                            logger.warn(f"Could not launch pod for job {self.instance.log_format}. Exceeded quota.")
+                            self.update_model(pk, status='pending')
+                            schedule_task_manager()
+                            return
+
```

Comment:
If it is not an exceed quota message, I think we should at least drop a log here so we can know which other issues we may face in the future.

Reply:
see: https://github.com/ansible/awx/pull/4922/files#r331744104